ledger/src/blockstore_processor.rs

```diff
@@ -344,6 +345,7 @@ pub fn process_blockstore(
         &opts.frozen_accounts,
         opts.debug_keys.clone(),
         Some(&crate::builtins::get(genesis_config.cluster_type)),
+        opts.bpf_jit,
```

Comment:
I think we can avoid plumbing `bpf_jit` though the Bank and InvokeContext entirely.  

Pass `opts.bpf_jit` as a new parameter into `crate::builtins::get()`, then select either
https://github.com/solana-labs/solana/blob/008b56e535ae2d94906b0d90e78e256cd893b881/ledger/src/builtins.rs#L12
or a new `solana_bpf_loader_with_jit_program!()`

Reply:
Don't the programs define which loader they need? In other words would the new loader work with the existing programs?

> Programs built for older loaders must use a matching entrypoint
> version.  An example is `bpf_loader_deprecated.rs` which requires
> `entrypoint_deprecated.rs`.

Reply:
Ah sorry so I was suggesting that `solana_bpf_loader_program` and `solana_bpf_loader_with_jit_program` have the same program id, and only one is added to the builtins based on the state of the `bpf_jit` flag

Reply:
I think I implemented it, but you should check if it is what you wanted.

Reply:
@mvines Circling back on this, it's kinda funky to define two builtins in order to support a feature especially since we'll have to add two builtins for each new loader we add (like the upgradeable).  What do you think about passing feature set into the bank with an existing configuration (for example jit enabled or not) and then if a feature account enabled a specific feature that would override.  This would allow for future features to be communicated into the runtime from the command line

Reply:
I think we should use a different mechanism than FeatureSet.   FeatureSet is supposed to be for things that would affect deterministic execution across all nodes in the cluster, and is currently based solely on on-chain state.

It's probably time to bundle these,
https://github.com/solana-labs/solana/blob/1d267eae6b82222027780280f8141710260c6123/runtime/src/bank.rs#L868-L870
into a `BankConfig` or `RuntimeConfig` struct that we can easily extend without an annoying plumbing effort on every new addition.  The JIT flag could then go in there?

Reply:
Yeah, sgtm, a different mechanism is more appropriate.